wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -20,7 +20,7 @@
         <div class="left">
             <div class="col header-title">
                 <h1>{% if icon %}{% icon name=icon class_name="header-title-icon" %}{% endif %}
-                    {{ title }}{% if subtitle %} <span>{{ subtitle }}</span>{% endif %}</h1>
+                    {{ title }}{% if subtitle %} <span class="subtitle">{{ subtitle }}</span>{% endif %}</h1>
```

Comment:
@thibaudcolas / @phildexter I'd like to propose we put the subtitle in its own line. 

That change does not exist in this PR but just thinking about long subtitles and how they really are not part of the title. 

The dashboard/ home page has a similar text but it is intentionally outside the h1. 

What do you think?


#3236 for example.

Reply:
actually, I think that this content should be inline here but maybe we need a new block that sits under the header, like a page description, which would be more suitable for the 'x of y count' content in form listings and other listings.